### ISSUE TITLE
Unable to manage connections

### ISSUE THREAD

**Issue by oliversalzburg** [pgmodeler/pgmodeler#835]
After upgrading to 8.2.1-beta1, all my settings were lost. When I now try to configure a connection, the dialog closes after finishing the configuration, but no actual connection is stored.

**Comment by oliversalzburg** [comment]
I was able to fix this by changing my `connections.conf` to:
```
<?xml version="1.0" encoding="UTF-8" ?>
<!--
  CAUTION: Do not modify this file directly on it's code unless you know what you are doing.
           Unexpected results may occur if the code is changed deliberately.
-->
<connections>
	<connection alias="local-db" host="localhost" port="5432" dbname="postgres"
		user="postgres" password="postgres" connection-timeout="2"
		sslmode="disable"
/>

</connections>
```

**Comment by oliversalzburg** [comment]
So, will this be fixed *ever*?

**Comment by rkhaotix** [comment]
What is the problem besides you may forgetting to click "Add" in connection
form before click apply?

Em 25 de ago de 2016 10:51, "Oliver Salzburg" <notifications@github.com>
escreveu:

**Comment by oliversalzburg** [comment]
Well, it would be nice if stored connections would persist between upgrades. 

There also seems to be a prompt that asks you if you want to save the entered data as a new connection when you click **Apply**, but it only seems to appear half the time I open the dialog.

Then the **Add** button isn't always enabled, even though I entered data.
![image](https://cloud.githubusercontent.com/assets/1658949/17974208/6e70ea26-6ae5-11e6-9c51-952bcb83372c.png)

Some of these might just as well be problems only introduced because previous `connections.xml` files could not be loaded. It is very frustrating to deal with, that's for sure.

That being said, I'm sadly not using the latest version, because I had a pretty major issue with 0.8.2, about which I sent a bug report via email (SQL export broken after upgrading to 0.8.2) 2 months ago and I never got a reply.

**Comment by rkhaotix** [comment]
Okaaay.

The connections were persisting between upgrades. But a change in connection's structure broke the process. But this is already fixed in 0.8.2. 

About the dialog not being shown I 'll investigate,  seems a small bug. And the " add" button not being enabled it is because you're not filling the minimum required fields: conn. alias, conn. db, host/port, user and password. 

About the issue in 0.8.2 you can fix it easily just look here #845. And about not getting a response for your email, I'm currently unable to use my computer in order to analyze reports because I am recovering from a surgery in the shoulder. Soon I will be back to the development. 

Finally, I try my best to keep users informed about bugs, give them support and take care of the code but I know that is not a simple task for a single developer. :(

**Comment by oliversalzburg** [comment]
Oh, sorry for being such a bitch :P I hope you recover soon and feel well in the meantime. Thanks for explaining the process to me, I found it very confusing when all of this happened and I was 100% sure that I was able to add connections with only those 4 fields filled out at some point.

We updated through the alphas and betas of 0.8.2 because we really needed the 9.5 support. Then when the final was released, the SQL export was broken :( I'm not 100% on the specifics anymore, there is probably a ticket about it by now. IIRC some tables were exported with a trailing comma or missing columns. I can repro the issue and post a ticket here if it helps.

Cheers
